### DISCUSSION TITLE
Couple of small bugs

### DISCUSSION THREAD

**Discussion by falOn-Dev** [BroncBotz3481/YAGSL-Example#86] [Q&A] [answered]
I just finished my first opportunity to test out this library, and I mainly noticed two problems with autonomous, the first one is less of a problem and more of an inconvenience, but when using the FollowTrajectory command the left and right side of the robot kind of go out of sync? like it wiggles forwards if that makes sense. I'm using PID values from this project as well and I figure its a PID issue.

The second issue could very well have something to do with my project, but when I try to schedule an auto routine that has more than just the path command, the robot code crashes/restarts, I included one of the example routines below if it helps.

```java
        public Command testDrive(){
                return new SequentialCommandGroup(
                        new HomingCommand(m_armPivotSubsystem, m_armTelescopeSubsystem),
                        new SetPositionCommandGroup(m_middleRowAngle, m_middleRowExtension),
                        new WaitCommand(0.1),
                        new HomingCommand(m_armPivotSubsystem, m_armTelescopeSubsystem),
                        new FollowTrajectory(drivebase, Constants.Paths.testDrive, true)
                );
        }
```

**Comment by thenetworkgrinch** [comment]
I am pretty sure it crashes because youre supposed to use (the now deprecated) waypoints in PathPlannerLib.

**Reply by falOn-Dev** [reply]
I'm not sure what you mean by that? The trajectory being passed in is from the .loadPath() method, using a path file from the path planner app

**Reply by thenetworkgrinch** [reply]
I have an example somewhere which shows how to use waypoints so pathplannerlib calls the commands. I think i remember this being an issue with it.

**Reply by falOn-Dev** [reply]
Do you happen to know which branch it's in? if not I'll just dig through a bit

**Reply by falOn-Dev** [reply]
Annoyingly pathplanner's wiki only contains information for their newest release which only works for the WPILIB 2024 beta, is this project compatible with that beta?

**Reply by thenetworkgrinch** [reply]
No, that would be unfair for teams to try to figure that out, however everything should work

**Comment by falOn-Dev** [comment, marked as answer]
Just so I can close this, we've since figured out the pathplanner crashing has nothing to do with yagsl or pathplanner and was a bug with our arm code
